src/commands/PublishCommand.js

```diff
@@ -590,6 +590,21 @@ export default class PublishCommand extends Command {
       changedFiles.push(packageJsonLocation);
     });
 
+    if (this.options.conventionalCommits) {
+      if (!this.repository.isIndependent()) {
```

Comment:
I probably would have combined this into one compound conditional (`this.options.conventionalCommits && !this.repository.isIndependent()`), but I am okay with this.